_inc/lib/debugger/class-jetpack-cxn-tests.php

```diff
@@ -91,11 +91,34 @@ public static function increase_timeout() {
 	protected function test__check_if_connected() {
 		$name = __FUNCTION__;
 		if ( $this->helper_is_jetpack_connected() ) {
-			$result = self::passing_test( $name );
+			$result = self::passing_test(
+				$name,
+				__( 'Test passed!', 'jetpack' ),
+				__( 'Your site is connected to Jetpack', 'jetpack' ),
+				sprintf(
+					'<p>%s</p><p>%s</p>',
+					__( 'A healthy connection ensures Jetpack essential services are provided to your WordPress site, such as Stats and Site Security.', 'jetpack' ),
+					__( '✅ Your site is connected to Jetpack.', 'jetpack' )
```

Comment:
Maybe we should pull the emoji out of the translation and use a placeholder instead? I am not sure translators / translator tools can all handle adding emoji.

Reply:
Agreed — ideally this should reuse a Dashicon instead of being an emoji:

Green/Healthy/Good:

```
<span class="dashicons pass"><span class="screen-reader-text">Passed</span></span>
```

Read/Error/Bad:

```
<span class="dashicons fail"><span class="screen-reader-text">Error</span></span>
```